CHANGELOG.md

```diff
@@ -1152,176 +1152,177 @@ candidates.
 
 ## 3.0.9
 
-  - [#479](https://github.com/paper-trail-gem/paper_trail/issues/479) - Deprecated
-    `originator` method in favor of `paper_trail_originator` Deprecation warning
-    informs users that the `originator` of the methods will be removed in
-    version `4.0`. (Backported from v4)
-  - Updated deprecation warnings for `Model.paper_trail_on` and
-    `Model.paper_trail_off` to have display correct version number the methods
-    will be removed (`4.0`)
+- [#479](https://github.com/paper-trail-gem/paper_trail/issues/479) - Deprecated
+  `originator` method in favor of `paper_trail_originator` Deprecation warning
+  informs users that the `originator` of the methods will be removed in
+  version `4.0`. (Backported from v4)
+- Updated deprecation warnings for `Model.paper_trail_on` and
+  `Model.paper_trail_off` to have display correct version number the methods
+  will be removed (`4.0`)
```

Comment:
Please do not change unrelated parts of the CHANGELOG. I'm guessing your IDE reformatted it automatically, but I don't want to read through 300 lines of diffs to confirm that.